packages/api/tests/endpoints/collections/update-collection.js

```diff
@@ -287,14 +292,14 @@ test.serial('PUT replaces an existing collection in all data stores with correct
   t.is(actualPgCollection.updated_at.getTime(), updatedEsRecord.updatedAt);
 });
 
-test.serial('PUT creates a new record in RDS if one does not exist  and sends an SNS message', async (t) => {
-  const knex = t.context.testKnex;
+test.serial('PUT creates a new record in Dynamo if one does not exist and sends an SNS message', async (t) => {
```

Comment:
Why did this test change from testing PostgreSQL to DynamoDB?

Reply:
Functionally the logic of the endpoint has changed.     

Prior to these updates the put endpoint would check *Dynamo* for existence, thus it was impossible to update a non-existent record in Dynamo, however it *was* the intended logic that an update happen to RDS even if there was no record. 

Moving the existence check from Dynamo *to* Postgres reverses the premise of this test as implemented in the code logic.      Given the discussed requirements of this (and related) tickets, I assumed that was the intended behavior.

For these endpoint updates to make the Postgres have primacy do we want to fail if it's missing from *either* Dynamo *or* RDS, or some other divergent behavior (e.g. only update the record if it exists)?     I think my vote is 'no', but I'm very willing to hear out commentary to the contrary.

Reply:
No I think that all makes sense and I don't think any further change is required here